src/schedule/Schedule.ts

```diff
@@ -133,41 +139,41 @@ export class Schedule extends LogEmitter {
 
   /**
    * Stops a scheduled job and removes it from the schedule (but not from the database).
-   * Does not check whether a job exists.
-   *
-   * If not provided with a job name, all scheduled jobs are canceled.
+   * Does nothing if no job with the given name exists.
    *
    * @param name the job to cancel
    */
-  public cancel(name?: string): void {
-    if (name) {
-      this.stop(name);
-      this.logger.debug('cancel', { name });
-      delete this.jobs[name];
-      return;
-    }
+  public cancelJob(name: string): void {
+    this.stopJob(name);
+    this.logger.debug('cancel', { name });
+    delete this.jobs[name];
+  }
 
+  /**
+   * Stops all scheduled jobs and removes them from the schedule (but not from the database).
+   */
+  public cancel(): void {
```

Comment:
I left the names short since `schedule.cancel()` already reads like "cancel the whole schedule => all jobs will be canceled" to me, but we can certainly discuss this if you see it different.

Reply:
Sounds good to me! We can keep it as is, then